Makefile

```diff
@@ -171,12 +171,26 @@ $(LIBTAILSCALE): Makefile android/libs $(LIBTAILSCALE_SOURCES) $(GOBIN)/gomobile
 
 libtailscale: $(LIBTAILSCALE)
 
-tailscale-new-debug.apk: $(LIBTAILSCALE)
+ANDROID_SOURCES=$(shell find android -type f -not -path "android/build/*" -not -path '*/.*')
```

Comment:
Since I had to add a new rule, I took this opportunity to make some improvements to our existing rule.